app/assets/stylesheets/_related.scss

```diff
@@ -3,16 +3,16 @@
   -webkit-box-sizing: border-box;
   -moz-box-sizing: border-box;
   box-sizing: border-box;
-  margin-top: 40px
+  margin-top: 40px;
 }
 
 .related {
   border-top: 10px solid $govuk-blue;
-  padding-top: 5px
+  padding-top: 5px;
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3069/files#r344819521)